appeals/web/src/server/appeals/appeal-details/site-visit/site-visit.service.js

```diff
@@ -59,8 +59,8 @@ export async function updateSiteVisit(
 			json: {
 				visitType,
 				...(visitDate && { visitDate }),
-				...(visitStartTime && { visitStartTime }),
-				...(visitEndTime && { visitEndTime })
+				...{ visitStartTime },
```

Comment:
if there's no risk of these being null or undefined, you should just be able to add them as shorthand object properties instead of using spread (eg. `json: { visitType, ...(visitDate && { visitDate }), visitStartTime, visitEndTime }`)